src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,18 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
+     * @param string|null $gender A gender of the last name should be generated
+     *     for. If the argument is skipped a random gender will be used.
+     * @return string Last name
+     */
+    public function lastName($gender = null)
+    {
+        return ($gender === static::GENDER_FEMALE) ?
```

Comment:
Instead of using a ternary statement here and repeating the method invocation to randomly select a last name, what do you think about introducing a temporary variable and then using a condition and early returns?

```suggestion
        $lastName = static::randomElement(static::$lastName);
        
        if (static::GENDER_FEMALE === $gender) {
            return $lastName . 'a';
        }
        
        return $lastName;
```

Also, would you mind adding two tests? Otherwise we’ll follow up!